### PR TITLE
John Doe profile data

### PR DESCRIPTION
**Pull request by Anko59**
Profile data for John Doe, dummy profile

### CODE REVIEW

File: aiden_app/static/images/user_images/cv/cv_John_Doe_default_profile.png
Comment (Arth-Leite) [inline aiden_app/static/images/user_images/cv/cv_John_Doe_default_profile.png:1]: Why is a png of the CV necessary ? Wouldn't PDF be better ?

**Review by Arth-Leite — APPROVED** [review]

File: aiden_app/static/images/user_images/cv/cv_John_Doe_default_profile.png
Comment (Anko59) [reply aiden_app/static/images/user_images/cv/cv_John_Doe_default_profile.png:1]: The pdf version is for download for the user, the png version is for display on the ui.

**Review by Smirkey — APPROVED** [review]
nice !

File: aiden_app/static/images/user_images/cv/cv_John_Doe_default_profile.png
Comment (Smirkey) [reply aiden_app/static/images/user_images/cv/cv_John_Doe_default_profile.png:1]: it's maybe better to display pdf, the user can copy/paste text, but fine for now I guess

Comment (Anko59) [reply aiden_app/static/images/user_images/cv/cv_John_Doe_default_profile.png:1]: Yeah, it seems to be super easy:
https://stackoverflow.com/questions/17784037/how-to-display-pdf-file-in-html
Can you of you write a ticket in the sprint board for this ?

Comment (Smirkey) [reply aiden_app/static/images/user_images/cv/cv_John_Doe_default_profile.png:1]: #2

**Pull request merged**